README.md

```diff
@@ -36,7 +36,7 @@ After that you can use one of the following Flyway commands:
 | `db clean`    | Drops all objects in the configured schemas                                  |
 | `db info`     | Prints the details and status information about all the migrations           |
 | `db validate` | Validates the applied migrations against the ones available on the classpath |
-| `db init`     | Creates and initializes the metadata table (existing database)               |
+| `db baseline` | Creates and initializes the metadata table (existing database)               |
```

Comment:
Since the command on Flyway is called `baseline` and not `init` I felt it was more accurately ran using that syntax